zoombot/zoombot/handler.go

```diff
@@ -117,10 +118,21 @@ func (h *Handler) zoomHandlerInner(msg chat1.MsgSummary) error {
 		h.ChatEcho(msg.ConvID, meeting.JoinURL)
 	case ZoomAPIError:
 		if err.Code == http.StatusTooManyRequests {
-			h.ChatEcho(msg.ConvID, err.Error())
-			return nil
+			if attempts > 5 {
+				h.ChatEcho(msg.ConvID, err.Error())
+				return nil
+			}
+			go func() {
+				attempts++
+				h.Debug("zoomHandlerInner: retrying attempt #%d: %v", attempts, err)
+				err := h.zoomHandler(msg, attempts)
+				switch err := err.(type) {
+				case nil, base.OAuthRequiredError:
+				default:
+					h.ChatErrorf(msg.ConvID, "zoomHandler unable to create meeting: attempt #%d %v", attempts, err)
+				}
+			}()
```

Comment:
Should return here, otherwise it returns the error to the handler:
<img width="513" alt="Screen Shot 2020-03-26 at 4 37 49 PM" src="https://user-images.githubusercontent.com/6353492/77694280-237b4680-6f80-11ea-8d5d-4159a84dbaca.png">

```suggestion
			}()
			return nil
```